src/helpers/tests/paths.test.ts

```diff
@@ -9,88 +9,100 @@ beforeEach(cleanup);
 afterAll(cleanup);
 
 describe("getFullModelsPaths", () => {
-  test("./dist/models", async () => {
+  test("./dist/models (ts)", async () => {
     setupFolderStructure("./dist/models", { index: false, js: true });
     // here the returned value should be an array containing paths of each individual schema
     const expected = [path.join(__dirname, "dist/models/user.js")];
 
-    let modelsPath = await paths.getFullModelsPaths(".", "js");
+    // undefined path
+    let modelsPath = await paths.getFullModelsPaths(undefined, "js");
     expect(modelsPath).toEqual(expected);
 
-    modelsPath = await paths.getFullModelsPaths("./src/helpers/tests/", "js");
-    expect(modelsPath).toEqual(expected);
-
-    modelsPath = await paths.getFullModelsPaths("./src/helpers/tests/dist", "js");
+    // empty string path
+    modelsPath = await paths.getFullModelsPaths("", "js");
     expect(modelsPath).toEqual(expected);
 
+    // defined path
     modelsPath = await paths.getFullModelsPaths("./src/helpers/tests/dist/models", "js");
     expect(modelsPath).toEqual(expected);
   });
 
-  test("./dist/models", async () => {
+  test("./dist/models (js)", async () => {
```

Comment:
Same as above, tests mention "ts" or nothing at all (I assume it defaults to "ts"), but the title says "js"...